book/chapters/chapter8/non-sequential_pipelines_and_tuning.qmd

```diff
@@ -284,13 +284,17 @@ bmr = benchmark(design)
 bmr$aggregate()[, .(learner_id, classif.ce)]
 ```
 
-In only a few lines of code, we took a weak learner and turned it into a powerful model that we can see is comparable to the implementation in `ranger::ranger`.
+In only a few lines of code, we took a weaker learner and turned it into a powerful model that we can see is comparable to the implementation in `ranger::ranger`.
 In the next section, we will look at a second example, which makes use of cross-validation within pipelines.
 
-### Stacking with "learner_cv" {#sec-pipelines-stack}
+### Stacking with `po("learner_cv")` {#sec-pipelines-stack}
 
-`r index('Stacking')` (@Wolpert1992) is another technique that can significantly improve model performance.
-The basic idea behind stacking is to use predictions from one model as features for a subsequent model to try to improve performance (@fig-pipelines-stacking).
+`r index('Stacking')` (@Wolpert1992) is another very popular ensembling technique that can significantly improve predictive performance.
+The basic idea behind stacking is to use predictions from multiple models (usually referred to as level 0 models) as features for a subsequent model (the level 1 model) which in turn combines these predictions (@fig-pipelines-stacking).
+A simple combination can be a linear model (maybe regularized if you have very many level 0 models), since a weighted sum of level 0 models is often plausible and good enough.
```

Comment:
```suggestion
A simple combination can be a linear model (possibly regularized if you have many level 0 models), since a weighted sum of level 0 models is often plausible and good enough.
```